tests/blockchain/test_blockchain.py

```diff
@@ -60,6 +56,7 @@
 
 log = logging.getLogger(__name__)
 bad_element = ClassgroupElement.from_bytes(b"\x00")
+replace = dataclasses.replace
```

Comment:
right, this was lgtm's suggestions. iiuc, using both `import ...` and `from  ... import ...` will cause the module to be imported twice. At least that was the impression lgtm's documentation gave me.

Do you suggest we instead import everything with the `from ... import ...` form?

Reply:
Module imports are global to the process, they are not `#include`s.  When you import in any form there is a check for the module you are requesting in `sys.modules`.  If there, that is used.  If not, it will import and add it to the `sys.modules` dict for next time.

In my projects I personally always did regular `import pytest` and then `pytest.raises()` and never used `from` imports since they throw out the namespacing and make it less obvious what the code is referencing.  For example, when you see `mkdir()` you think `os.mkdir()` not `chia.util.path.mkdir()` which wraps `pathlib.Path.mkdir(parents=True, exist_ok=True)`.  This tripped up several of us in reviews.  But sure, regular imports make for much longer references to things you have imported.

What should we do?  Until we decide we need to make a stricter policy I would just stick to either regular or from imports for any given top-level module being imported within a particular other module.    I think we've mostly been consistent about that, but apparently not quite completely.  Maybe isort can help with this in the future instead of lgtm.  But my interest in this is just to reduce the amount of thinking we have to do about how to add a new import from an existing module and not related to any performance factor.